docs/2.guide/2.directory-structure/1.pages.md

```diff
@@ -299,6 +299,10 @@ You may define a name for this page's route.
 
 You may define a path matcher, if you have a more complex pattern than can be expressed with the file name. See [the `vue-router` docs](https://router.vuejs.org/guide/essentials/route-matching-syntax.html#custom-regex-in-params) for more information.
 
+#### `clientOnly`
+
+Will wrap your page in [the Nuxt `<ClientOnly>` component](/docs/api/components/client-only) if you set `clientOnly: true` in your `definePageMeta`. This might be useful if you trying to import library that depends on `window` in page file.
```

Comment:
I'd word the "library" part a bit more general and give examples then. Something like

```suggestion
Will wrap your page in [the Nuxt `<ClientOnly>` component](/docs/api/components/client-only) if you set `clientOnly: true` in your `definePageMeta`. This might be useful if you trying to import library that has no support for server-side rendering, e.g. needs `document` or `window`.
```